### PR TITLE
ansible: replace yum-utils by dnf-utils

### PR DESCRIPTION
**Pull request by lebauce**
skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-k8s-tests
skip skydive-ppc64le-tests
skip skydive-functional-tests-ebpf
skip skydive-functional-hw-tests

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-create-vagrant-boxes

**Comment by lebauce** [comment]
run skydive-create-vagrant-boxes

**Comment by lebauce** [comment]
run  skydive-create-vagrant-boxes

File: contrib/ansible/roles/skydive_dev/tasks/main.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@
      - selinux-policy-devel
      - policycoreutils-devel
      - mock
-     - yum-utils
+     - dnf-utils
```
Comment (nplanel) [inline contrib/ansible/roles/skydive_dev/tasks/main.yml]: I'm not sure it would work on centos / rhel ?

**Comment by lebauce** [comment]
run skydive-create-vagrant-boxes

File: contrib/ansible/roles/skydive_dev/tasks/main.yml (hunk 1 above)
Comment (lebauce) [reply contrib/ansible/roles/skydive_dev/tasks/main.yml]: This is the `skydive_dev` role. We use for our CI machines and the vagrant boxes so we don't need to bother about CentOS and RHEL

**Comment by lebauce** [comment]
run skydive-create-vagrant-boxes

**Comment by lebauce** [comment]
run skydive-create-vagrant-boxes

**Comment by masco-enovance** [comment]
run skydive-create-vagrant-boxes

**Comment by lebauce** [comment]
run skydive-create-vagrant-boxes

**Comment by lebauce** [comment]
run skydive-create-vagrant-boxes

**Pull request merged**